src/main/java/baseball/GameManager.java

```diff
@@ -0,0 +1,86 @@
+package baseball;
+
+import baseball.generator.RandomNumberGenerator;
+import baseball.receiver.ConsoleInputReceiver;
+import baseball.receiver.InputReceiver;
+import baseball.number.GameNumber;
+import baseball.printer.ConsolePrinter;
+import baseball.printer.Printer;
+import baseball.result.Result;
+
+import java.util.InputMismatchException;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class GameManager {
+    private final int START_GAME = 1;
```

Comment:
상수는 private static final!
```suggestion
    private static final int START_GAME = 1;
```